api/src/utils/csv-utils/csv-config-utils.ts

```diff
@@ -204,26 +204,77 @@ export class CSVConfigUtils<StaticHeaderType extends Uppercase<string> = Upperca
     }
   }
 
+  /**
+   * Get the array-cell values from a CSV row.
+   *
+   * @param {StaticHeaderType} header - The header name
+   * @param {CSVRow} row - The CSV row
+   * @param {{ delimiter: string }} options - The options
+   * @return {(string[] | undefined)} - The array-cell values or undefined if none found
+   */
+  getArrayCellValue(header: StaticHeaderType, row: CSVRow, options: { delimiter: string }): string[] | undefined {
+    // Static header or dynamic header exact match
+    let cellValue = this.getCellValue(header, row);
+
+    if (!cellValue) {
```

Comment:
note: This block from 219 -> 224 is not need I don't think. `getCellValue` already handles the aliases

Reply:
fixed